src/assets.zig

```diff
@@ -364,17 +380,21 @@ pub fn loadWorldAssets(assetFolder: []const u8, blockPalette: *Palette, biomePal
 		arenaAllocator,
 		assetFolder,
 		&blocks,
-		&blocksMigrations,
+		&blockMigrations,
 		&items,
 		&tools,
 		&biomes,
+		&biomeMigrations,
 		&recipes,
 		&models,
 	);
 	errdefer unloadAssets();
 
-	migrations_zig.registerBlockMigrations(&commonBlocksMigrations);
-	migrations_zig.applyBlockPaletteMigrations(blockPalette);
+	migrations_zig.registerAll(.block, &blockMigrations);
+	migrations_zig.apply(.block, blockPalette);
+
+	migrations_zig.registerAll(.biome, &biomeMigrations);
+	migrations_zig.apply(.biome, biomePalette);
```

Comment:
One more thing to consider for the future:
If we directly apply them, do we really need to register and store them?

Reply:
Yeah, It came to my mind too. Technically we don't and even with current implementation we should free them instantly after using them. I will think about better design after I tackle item palette so I have full picture of the problem.